index.js

```diff
@@ -1,9 +1,13 @@
 const express = require('express');
 const app = express();
 const PORT = 4000;
+const admins = require('./controllers/admins');
```

Comment:
put the import (`const admins = require('./controllers/admins');`) at the top of the file, it is a convention